src/contracts/Limitation/Type.php

```diff
@@ -57,7 +57,7 @@ public function acceptValue(APILimitationValue $limitationValue);
     /**
      * Makes sure LimitationValue->limitationValues is valid according to valueSchema().
      *
-     * Make sure {@link acceptValue()} is checked first!
+     * Make sure {@see acceptValue()} is checked first!
```

Comment:
Avoid shouting at the reader:
```suggestion
     * Make sure {@see Type::acceptValue()} is checked first.
```

Reply:
Good point, thanks!